monai/transforms/croppad/dictionary.py

```diff
@@ -372,11 +373,19 @@ def __init__(
         super().__init__(keys, allow_missing_keys)
         self.cropper = SpatialCrop(roi_center, roi_size, roi_start, roi_end, roi_slices)
 
-    def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> Dict[Hashable, NdarrayOrTensor]:
+    def __call__(
+        self, data: Mapping[Hashable, Union[NdarrayOrTensor, Dict]]
+    ) -> Dict[Hashable, Union[NdarrayOrTensor, Dict]]:
         d = dict(data)
         for key in self.key_iterator(d):
             self.push_transform(d, key)
-            d[key] = self.cropper(d[key])
+            meta_key = PostFix.meta(key)
+            meta = d.get(meta_key, None)
+            if meta is None:
+                d[key] = self.cropper(d[key])
+            else:
+                d = backup_meta(d, key)
```

Comment:
I don't quite understand why call `backup_meta` here?
Maybe I misunderstood the usage of this function.

Thanks.